pkg/nvml/device.go

```diff
@@ -3254,6 +3398,34 @@ func (device nvmlDevice) SetNvlinkBwMode(setBwMode *NvlinkSetBwMode) Return {
 	return nvmlDeviceSetNvlinkBwMode(device, setBwMode)
 }
 
+func (l *library) DeviceGetNvLinkInfo(device Device) NvLinkInfoHandler {
+	return device.GetNvLinkInfo()
+}
+
+func (device nvmlDevice) GetNvLinkInfo() NvLinkInfoHandler {
+	return NvLinkInfoHandler{device}
+}
```

Comment:
Normally we would put an `V` at the end of this function (and put them below handler and V1,V2, etc. variants.

Reply:
It seems as if both the `_v1` and `_v2` types were added in 13.x along with a single `nvmlDeviceGetNvLinkInfo` function. This does *not* have the `V` suffix like other NEW versioned functions since I assume that this was implemented to be versioned from the start. 

I can do a more thourough analysis of these if you like.